{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -133,13 +128,87 @@ accept-no-yields-doc = false
 default-docstring-type = "google"
 
 [tool.pylint.design]
-max-locals=20
+max-locals = 20
 
 [tool.pylint.similarities]
-min-similarity-lines=10
+min-similarity-lines = 10
 
 [tool.bandit]
-exclude_dirs = ["venv",]
+
+
+
+
+{% if cookiecutter.versioning == "Python Semantic Release" %}
```

Comment:
To be honest I would leave this without if it shall not break anything.
If You want go with this if than {% raw %} is missing to escape e.g block_start_string = {%